### PR TITLE
Some profile string tweaks

### PR DESCRIPTION
**Pull request by RiseT**
### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * Nexus 5X, Android 8.0.0
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)
- [x] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in the commit message of my first commit

----------

### Description

**"Updating and encrypting profile" progress dialog**
- Swap title and body text (the shorter string as title)
- Shorten the title somewhat more (in order to prevent potential line wraps within the title for some languages; in line with other progress dialogs)
- Add missing ellipsis to the text body (per Material design guidelines)

**"Too long"message**
- Make string clearer (to translators and potentially to end users as well)

### CODE REVIEW

**Comment by RiseT** [comment]
Preliminarily closed.

File: res/values/strings.xml [hunk 1]
```diff
@@ -234,10 +234,10 @@
     <string name="CreateProfileActivity_your_profile_info">Your profile info</string>
     <string name="CreateProfileActivity_error_setting_profile_photo">Error setting profile photo</string>
     <string name="CreateProfileActivity_problem_setting_profile">Problem setting profile</string>
-    <string name="CreateProfileActivity_updating_profile">Updating profile</string>
-    <string name="CreateProfileActivity_updating_and_encrypting_profile">Updating and encrypting profile</string>
+    <string name="CreateProfileActivity_updating">Updating</string>
+    <string name="CreateProfileActivity_updating_and_encrypting_profile">Updating and encrypting profile...</string>
     <string name="CreateProfileActivity_profile_photo">Profile photo</string>
-    <string name="CreateProfileActivity_too_long">Too long</string>
+    <string name="CreateProfileActivity_profile_name_is_too_long">Profile name is too long</string>
```
Comment (moxie0) [inline res/values/strings.xml]: please revert this one

**Comment by RiseT** [comment]
@moxie0 Done

**Comment by RiseT** [comment]
As a sidenote, I've noticed that progress dialogs like this one got deprecated with API level 26. Perhaps replacing these with progress bars would make sense. These look pretty good imho.

https://developer.android.com/reference/android/widget/ProgressBar.html

**Comment by RiseT** [comment]
I guess this got superseded by https://github.com/WhisperSystems/Signal-Android/commit/4c2269175b6477c42f8aa6ec5cd4a376cc27a0f9, right?

**Comment by moxie0** [comment]
Yep the button turns into a progress bar now and then circular reveals

**Pull request closed without merging**